apidoc/Titanium/Gesture/Gesture.yml

```diff
@@ -56,8 +56,8 @@ properties:
         <Titanium.UI.FACE_DOWN>, 
         or <Titanium.UI.UNKNOWN>. 
         
-        On iOS, returns the current orientation of the *device*, regardless of the screen
-        orientation. To determine the current screen orientation on iOS, use 
+        On iOS, returns the current orientation of the *device*, regardless of the window
+        orientation. To determine the current window orientation on iOS, use 
         <Titanium.UI.orientation>. 
         
```

Comment:
This information applies to Windows Phone too. And this paragraph is referencing a deprecated API. How about we change it to the following?

```
On iOS and Windows, returns the current orientation of the *device*, regardless of the window orientation. To determine the current window orientation, use [Window.orientation](Titanium.UI.Window.orientation).
```